src/main/java/seedu/address/model/income/Income.java

```diff
@@ -0,0 +1,130 @@
+package seedu.address.model.income;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Set;
+
+import seedu.address.model.claim.Amount;
+import seedu.address.model.claim.Description;
+import seedu.address.model.commonvariables.Name;
+import seedu.address.model.commonvariables.Phone;
+import seedu.address.model.tag.Tag;
+
+
+
+
+/**
+ * Represents an Income in the Financial Planner.
+ */
+public class Income {
+
+    // Identity fields
+    private final Name name;
+    private final Phone phone;
+    private final Description description;
+
+    //Data fields
+    private final Amount amount;
+    private final Set<Tag> tags = new HashSet<>();
+    private final Date dateAdded;
+
+    /**
+     * Every field must be present and not null.
```

Comment:
Javadocs should be uniform with a present tense verb starting the javadoc

Reply:
Oh right, thank you!